html-tables/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Tables</title>
+</head>
+
+<body>
+  <h1>Student Grade Table</h1>
+  <table>
+
+    <tr>
+      <th colspan="1">Name</th>
+      <th colspan="2">Course</th>
+      <th colspan="3">Grade</th>
+    </tr>
+    <td>Grace Hopper</td>
```

Comment:
These poor `td` elements seem to be missing their `tr` around them. They're so lonely! :cry: